src/types/networks.ts

```diff
@@ -1,3 +1,9 @@
 export enum NetworkType {
   RSK,
 }
+
+declare global {
```

Comment:
This should be in a .d.ts file usually next to the react-app-env.d.ts, I would name either typings.d.ts or global.d.ts.
It would also be used for untyped dependencies, when necessary.

That way you separate types and global typing.

Reply:
Fixed in the latest commit.